user_docs/en/userGuide.t2t

```diff
@@ -311,6 +311,7 @@ If you have add-ons already installed there may also be a warning that incompati
 Before you're able to press the Continue button you will have to use the checkbox to confirm that you understand that these add-ons will be disabled.
 There will also be a button present to review the add-ons that will be disabled.
 Refer to the [incompatible add-ons dialog section #incompatibleAddonsManager] for more help on this button.
+You are able to re-enable incompatible add-ons at your own risk.

```

Comment:
```suggestion
You are able to re-enable incompatible add-ons from within the Add-on Store dialog after installation at your own risk.
```

Reply:
I agree with @michaelDCurran regarding sorting columnas. I'd prefer that a combo box would be included and I think that the current shortcuts can be removed from the user guide.

Reply:
+1